quarkus-integration/quarkus/integration-test/src/test/java/ai/timefold/solver/quarkus/it/TimefoldTestResourceTest.java

```diff
@@ -27,4 +27,17 @@ void solveWithSolverFactory() throws Exception {
                         "0hard/5soft"));
     }
 
+    @Test
+    @Timeout(600)
+    void solveWithTimeOverride() throws Exception {
```

Comment:
I got this error:

    [ERROR] ai.timefold.solver.quarkus.it.TimefoldTestResourceTest.solveWithTimeOverride -- Time elapsed: 10.98 s <<< FAILURE!
    java.lang.AssertionError:
    1 expectation failed.
    Response body doesn't match expectation.
    Expected: is "0hard/5soft,0.50"
    Actual: 0hard/5soft,0,50

Likely a locale difference between my computer and yours. Please make the test locale-independent.